requirements/test.in

```diff
@@ -6,7 +6,8 @@
 coverage
 ddt
 mock
-nose
+pytest
```

Comment:
We don't need to `pytest` in requirements because it is base requirement of `pytest-cov`
https://github.com/pytest-dev/pytest-cov/blob/master/setup.py#L125